pkgs/by-name/nx/nxwitness-client/package.nix

```diff
@@ -0,0 +1,119 @@
+{
+  stdenv,
+  lib,
+  autoPatchelfHook,
+  buildFHSEnv,
+  dpkg,
+  fetchurl,
+  glib,
+  gst_all_1,
+  libGL,
+  libgudev,
+  libudev-zero,
+  libxcb,
+  libxkbfile,
+  libxml2,
+  libxslt,
+  openal,
+  qt6Packages,
+  wayland,
+  xcbutilimage,
+  xcbutilkeysyms,
+  xcbutilrenderutil,
+  xcbutilwm,
+}:
+let
+  version = "6.0.3";
+  build = "40736";
+
+  libxml2_13 = libxml2.overrideAttrs rec {
```

Comment:
I think we just yesterday added knownVulnerabilities to this version https://github.com/NixOS/nixpkgs/blob/e6ff9e9481fdb7877f3be5991d3f583d4fad3bbf/pkgs/applications/networking/remote/citrix-workspace/generic.nix#L93-L104

Reply:
Added it here too.